### PR TITLE
Ship Review: VPN Website Exclusions

### PR DESCRIPTION
**Pull request by diegoreymendez**
Task/Issue URL: https://app.asana.com/0/1206580121312550/1208287874374302/f

## Description

Releases VPN Website Exclusions to the public:
- Remove the internal user limitations and the feature flag.
- Addressed copy / design / code review changes.

## Resources

- [Figma file with latest designs and copy](https://app.asana.com/0/1175293949586521/1208034562783571)

## Testing

Test VPN Website Exclusions.

**Definition of Done**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)

### CODE REVIEW

File: DuckDuckGo/Common/Localizables/UserText+NetworkProtection.swift [hunk 1]
```diff
@@ -199,7 +199,7 @@ extension UserText {
     // "vpn.location.title" - Location section title in VPN settings
     static let vpnLocationTitle = "Location"
     // "vpn.excluded.sites.title" - Excluded Sites title in VPN settings
-    static let vpnExcludedSitesTitle = "Excluded Sites"
```
Comment (diegoreymendez) [inline DuckDuckGo/Common/Localizables/UserText+NetworkProtection.swift:202]: Several copy updates based on lastest figma.

File: DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift [hunk 2]
```diff
@@ -91,27 +91,24 @@ final class VPNPreferencesModel: ObservableObject {
...
     private let settings: VPNSettings
     private let pinningManager: PinningManager
-    private let internalUserDecider: InternalUserDecider
     private var cancellables = Set<AnyCancellable>()
 
     init(vpnXPCClient: VPNControllerXPCClient = .shared,
          settings: VPNSettings = .init(defaults: .netP),
          pinningManager: PinningManager = LocalPinningManager.shared,
-         defaults: UserDefaults = .netP,
-         internalUserDecider: InternalUserDecider = NSApp.delegateTyped.internalUserDecider) {
```
Comment (diegoreymendez) [inline DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift:101]: Also removing the internal user limitation for the feature, and the associated feature flag.

File: LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/Font+NetworkProtection.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+extension Font {
```
Comment (diegoreymendez) [inline LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/Font+NetworkProtection.swift:21]: Sharing fonts so they can be used across NetP views.

File: LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/View+NetworkProtection.swift [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        colorScheme == .light ? Double(0.6) : Double(0.5)
+    }
+
+    static func title(colorScheme: ColorScheme) -> Double {
+        colorScheme == .light ? Double(0.84) : Double(0.85)
+    }
+}
+
+extension View {
+    func applyConnectionStatusDetailAttributes(colorScheme: ColorScheme) -> some View {
```
Comment (diegoreymendez) [inline LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/View+NetworkProtection.swift:48]: Sharing attributes so they can be used across NetP views.

File: LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/SiteTroubleshootingView/SiteTroubleshootingInfo.swift [hunk 5]
```diff
@@ -20,12 +20,12 @@ import AppKit
 import Foundation
 
 public struct SiteTroubleshootingInfo {
-    public let icon: NSImage
+    public let icon: NSImage?
     public let domain: String
     public let excluded: Bool
 
     public init(icon: NSImage?, domain: String, excluded: Bool) {
-        self.icon = icon ?? NSImage(systemSymbolName: "globe", accessibilityDescription: nil)!
```
Comment (diegoreymendez) [inline LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/SiteTroubleshootingView/SiteTroubleshootingInfo.swift:28]: We won't show any icon if the favicon is missing.

File: DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift (hunk 2 above)
Comment (samsymons) [reply DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift:101]: In the future we could likely use an internal feature flag, and then just flip it to live when we're ready. (That would also allow for rolling it out incrementally)

File: LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/Font+NetworkProtection.swift (hunk 3 above)
Comment (samsymons) [reply LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/Font+NetworkProtection.swift:21]: I was going to suggest that [DesignResourcesKit](https://github.com/duckduckgo/DesignResourcesKit) may be useful here, but I realized it isn't used on macOS. Never mind :)

**Review by samsymons — APPROVED** [review]
I can't speak for whether there will be more ship review changes coming, but the diff for these looks good to me, happy to approve it. Thanks @diegoreymendez!

**Pull request merged**